doc/cookbook/source/examples/neural_nets/feedforward_net_regression.rst

```diff
@@ -0,0 +1,45 @@
+==================================
+Feedforward Network for Regression
```

Comment:
can you pls send a separate patch that adds the nn examples under regression and classification in the meta examples as well (so they appear multiple times), also for the gp examples.